theme/templates/resource-landing-page/citation.html

```diff
@@ -169,7 +169,7 @@
                 </div>
 
                 {% if is_owner_user and cm.resource_type == "NetcdfResource" and not cm.raccess.public %}
-                    <div class="pull-left label-public">Note that making the resource public may take a little extra time to update
+                    <div class="label-public">Note that making the resource public may take a little extra time to update
```

Comment:
It was added as part of refactoring of inline css: https://github.com/hydroshare/hydroshare/commit/3b577cf7affb5bcf0b41ff07b8752bd283b1231f